java/test/src/main/java/io/ray/test/CrossLanguageInvocationTest.java

```diff
@@ -227,6 +312,22 @@ public static String returnInputString(String s) {
     return (byte[]) res.get();
   }
 
+  public static Object throwException() {
+    return 1 / 0;
+  }
+
+  public static Object raiseJavaException() {
+    ObjectRef<Object> res = Ray.task(
+        PyFunction.of(PYTHON_MODULE, "py_func_java_throw_exception", Object.class)).remote();
+    return res.get();
+  }
+
+  public static Object raisePythonException() {
```

Comment:
```suggestion
  public static Object raisePythonError() {
```

I'm not sure which one is better, `raise` or `throw`. But we'd better always use `raise`/`error` and `throw`/`exception` as pairs. Same for Python code.

Reply:
The official expression of the `Python` exception: 

> The try … except statement has an optional else clause, which, when present, must follow all except clauses. It is useful for code that must be executed if the try clause does not `raise an exception`. https://docs.python.org/3/tutorial/errors.html

But, many built-in exceptions are named XXXError: https://docs.python.org/3/library/exceptions.html#Exception

I think the name `raisePythonException` is pythonic enough.